src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -15120,7 +15064,6 @@ void VRFTerminalUnitEquipment::CalcVRFSuppHeatingCoil(EnergyPlusData &state,
     Real64 constexpr Acc(1.e-3); // Accuracy of solver result
 
     // local variable declaration:
-    int SolFla;              // Flag of solver, num iterations if >0, else error index
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:15123]:(style),[variableScope],The scope of the variable 'SolFla' can be reduced.